Tryouts/Prototypes/Shell/MainWindow.xaml.cs

```diff
@@ -12,18 +12,32 @@
 //  * and limitations under the License.
 //  */
 
+using Manifest;
 using System;
 using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Text;
+using System.Threading.Tasks;
 using System.Windows;
 using System.Windows.Controls;
-using Manifest;
+using System.Windows.Data;
+using System.Windows.Documents;
+using System.Windows.Input;
+using System.Windows.Media;
+using System.Windows.Media.Imaging;
+using System.Windows.Navigation;
+using System.Windows.Shapes;
+using System.Windows.Controls.Ribbon;
+using System.Windows.Controls.Ribbon.Primitives;
 
 namespace Shell
 {
     /// <summary>
     /// Interaction logic for MainWindow.xaml
     /// </summary>
-    public partial class MainWindow : Window
+    public partial class MainWindow : RibbonWindow
```

Comment:
This is not a critique for this PR, or the current state and goals, just a note that in the future, we probably should devise a more flexible solution for adding visual features than inheritance.

Reply:
This is fine for short term demoing purposes, but we will probably not have a ribbon control on the main window by default. As discussed earlier (and I mean years ago), we should provide a very minimalistic shell at first, concentrating on docking and layout persistence, and maybe later we can start adding features like a ribbon chrome.